.vitepress/sidebars/develop.ts

```diff
@@ -280,6 +280,30 @@ export default [
       },
     ],
   },
+  {
+    text: "develop.porting",
+    collapsed: true,
+    items: [
+      {
+        text: "develop.porting.introduction",
+        link: "/develop/porting/introduction",
+      },
+      {
+        text: "develop.misc.migrating_mappings",
```

Comment:
```suggestion
        text: "develop.porting.migrating_mappings",
```

this change must be applied in (at least) six places: three in this file, three in sidebar_translations.json